geode-junit/src/main/java/org/apache/geode/test/internal/rules/gfsh/GfshRule.java

```diff
@@ -31,17 +31,17 @@
 import org.junit.rules.ExternalResource;
 import org.junit.rules.TemporaryFolder;
 
-import org.apache.geode.management.internal.cli.commands.StatusLocatorRealGfshTest;
-import org.apache.geode.management.internal.cli.util.CommandStringBuilder;
-import org.apache.geode.test.dunit.rules.RequiresGeodeHome;
+import org.apache.geode.test.internal.rules.RequiresGeodeHome;
+
 
 /**
  * The {@code GfshRule} allows a test to execute Gfsh commands via the actual (fully-assembled) gfsh
- * binaries. For a usage example, see {@link StatusLocatorRealGfshTest}. Each call to
- * {@link GfshRule#execute(GfshScript)} will invoke the given gfsh script in a forked JVM. The
- * {@link GfshRule#after()} method will attempt to clean up all forked JVMs.
+ * binaries. Each call to {@link GfshRule#execute(GfshScript)} will invoke the given gfsh script in
+ * a forked JVM. The {@link GfshRule#after()} method will attempt to clean up all forked JVMs.
  */
 public class GfshRule extends ExternalResource {
+  private static final String SINGLE_QUOTE = "\"";
+
```

Comment:
This choice of name is misleading to me. I think what it is intended to convey is "a string containing a single double quote character" (ASCII 0x22) NOT "a string containing a```'``` character" (ASCII 0x27).

Reply:
I originally copied this from CommandStringBuilder since that class is not available in geode-junit, but I agree that the name is extremely misleading. I'll change the name in both places.